### DISCUSSION TITLE
Jupyterbook with R engine

### DISCUSSION THREAD

**Discussion by StraightDraw** [executablebooks/meta#329] [Q&A] [answered]
How do I create a jupyterbook using an R engine? I'm not a complete noobie. I have created jupyterbooks using:

[**Python**](https://straightdraw.github.io/stats/Introduction.html),
[**MyST only**](https://straightdraw.github.io/Geometry/Introduction.html), and a
[**MATLAB** engine](https://straightdraw.github.io/Linear/Introduction.html)

I cannot get notebooks that use an R engine to compile into a jupyterbook using the `jb build content` command. I have searched this forum but have been unable to locate any resources discussing it (R is a problematic search term).

**Comment by chrisjsewell** [comment, marked as answer]
If I understand correctly perhaps you are looking for: https://irkernel.github.io/

**Reply by StraightDraw** [reply]
Thanks, now I'm no longer getting an 

jupyter_client.kernelspec.NoSuchKernel

error. However, it's still not compiling. I will work on it and report back.

**Reply by StraightDraw** [reply]
I don't understand where I've screwed up. I installed IRKernel as shown at link. Attached new error log.

[sphinx-err-7xhkd69c.log](https://github.com/executablebooks/meta/files/6336478/sphinx-err-7xhkd69c.log)
